tests/kernel/mem_protect/mem_protect/src/kobject.c

```diff
@@ -65,7 +65,10 @@ static void syscall_invalid_kobject_user_part(void *p1, void *p2, void *p3)
 }
 
 /**
- * @brief Test if a syscall can take a different type of kobject
+ * @brief Test syscall can take a different type of kobject
+ *
+ * @details Test syscall can take a different type of kobject and syscall will
```

Comment:
tip: you can actually dtop the @details command, anything at this level is considered details by default.